bin/git/git.js

```diff
@@ -226,22 +232,43 @@ export async function getBaselineCommits(client, { branch, ignoreLastBuildOnBran
   const initialCommitsWithBuilds = [];
   const extraBaselineCommits = [];
 
-  // Don't do any special branching logic for builds on `HEAD`, this is fairly meaningless
-  // (CI systems that have been pushed tags can not set a branch)
+  // Add the most recent build on the branch as a baseline (unless the user opts out), or it is newer than this commit(?)
+  // @see https://www.chromatic.com/docs/branching-and-baselines#rebasing
+  // [Don't do any special branching logic for builds on `HEAD`, this is fairly meaningless
+  // (CI systems that have been pushed tags can not set a branch)]
   if (
     branch !== 'HEAD' &&
     !ignoreLastBuildOnBranch &&
     lastBuild &&
     lastBuild.committedAt <= committedAt
   ) {
     if (await commitExists(lastBuild.commit)) {
+      debug(`Adding last branch build commit ${lastBuild.commit} to commits with builds`);
```

Comment:
Did you test these debug statements? I'm not sure setupDebug still works. I typically use `ctx.log.debug()`.